src/App.less

```diff
@@ -0,0 +1,4 @@
+.app-flex-container {
+  // need to use !important to overwrite .ant-layout
+  min-height: 100vh !important;
```

Comment:
How often will we have to use `!important`? I've been told to always avoid it

Reply:
Funny story...
Behind the scenes, AntD is loading its Layout css class. Due to the order in which the css is processed, our less files are not taking priority over the AntD default .ant-layout class.

Now there is a better way to do this, which is to implement the .ant-layout class ourselves as shown in documentation. I feel like this is just worse. Its literally copy pasting code to overwrite a single attribute (min-height). So I used !important instead.

I was afraid this might happen, and I wonder how this will affect our other "custom" css components. I imagine it would only be more difficult to customize other AntD components (without cascading global changes to all other AntD components).